src/app/about-us/page.tsx

```diff
@@ -1,22 +1,27 @@
 import React, { FC } from "react";
 import AboutHero from "@/components/about-us/AboutHero";
-import AboutOurMission from "@/components/about-us/AboutOurMission";
-import BoardMembers from "@/components/about-us/BoardMembers";
-import MissionStatement from "@/components/about-us/MissionStatement";
-import Timeline from "@/components/about-us/Timeline";
+import AboutOurMissionVision from "@/components/about-us/AboutOurMissionVision";
 import missionStatementData from "@/data/about-us";
-import PageHead from "@/components/about-us/PageHeader";
+import { Separator } from "@radix-ui/themes";
+import TimelineItem from "@/components/about-us/Timeline";
 
 const AboutUsPage: FC = () => {
-  const { missionStatement, aboutOurMission, timeline } = missionStatementData;
+  const { aboutOurMission, aboutOurVision, timeline } = missionStatementData;
 
   return (
-    <main className="">
+    <main>
       <AboutHero />
-      <MissionStatement missionStatement={missionStatement} />
-      <BoardMembers />
-      <AboutOurMission content={aboutOurMission} />
-      <Timeline items={timeline} />
+      <div className="flex mt-10 md:mt-20 bg-gray-50 h-full ">
+        <div className="flex flex-col py-[47px] lg:flex-row justify-center items-center gap-0 lg:gap-10 xl:gap-20 px-[50px] md:px-[117px] h-full">
```

Comment:
See notes above about using RadixUI's Flex component.

**question:** Why such specific pixel values? (ex: py-[47px] or md:px-[117px]).  Would prefer to use the nearest RadixUI size value for margin / padding to keep the code simple, instead of trying to match the designs in a pixel-perfect manner.